context/app/static/js/components/detailPage/related-entities/RelatedEntitiesTabs/style.js

```diff
@@ -11,11 +13,24 @@ const StyledTabPanel = styled(TabPanel)`
 
 const StyledTabs = styled(Tabs)`
   flex: none;
+
+  span {
+    display: contents;
+  }
+
+  &&& svg {
+    margin-bottom: 0;
+  }
```

Comment:
This is slightly different to what I suggested, but that's okay. 

I generally prefer to style components directly (creating separate styled components as necessary) instead of leveraging css selectors. If we do have to use css selectors, like we do here, I try to make sure we limit the scope of the css as much as possible.

In this case, we are trying to apply styles to span and svg elements inside each `Tab`. I would apply the styles in a new styled `Tab`.

Reply:
Here's a brief discussion about the topic https://www.robinwieruch.de/styled-components/#singlemultiple-styled-components.

Reply:
It worked :)